docs/data-sources/project_policy.md

```diff
@@ -21,12 +21,14 @@ description: |-
 
 ### Read-Only
 
+- **continuous_deployment_default** (Boolean) Redeploy on every push to the git branch default value
 - **disable_destroy_environments** (Boolean) Disallow destroying environment in the project
 - **id** (String) id of the policy
 - **include_cost_estimation** (Boolean) Enable cost estimation for the project
 - **number_of_environments** (Number) Max number of environments a single user can have in this project, 0 indicates no limit
 - **number_of_environments_total** (Number) Max number of environments in this project, 0 indicates no limit
 - **requires_approval_default** (Boolean) Requires approval default value when creating a new environment in the project
+- **run_pull_request_plan_default** (Boolean) Run Terraform Plan on Pull Requests for new environments targeting their branch default value
```

Comment:
why they not near each other - both of them belong trigger policy. 
Plus I think id should be the first one

Reply:
its generated